Controllers/CustomerController.cs

```diff
@@ -34,16 +37,23 @@ public ActionResult Get(int id)
         [HttpPost]
         public ActionResult Post([FromBody] CustomerCreate body)
         {
+            if (!User.IsInRole("admin") && body.Role == "admin")
+            {
+                return Forbid();
+            }
+
```

Comment:
Could you move this to the CustomerService

Reply:
also, perhaps I'm overhinking this a good bit, but I am not sure how people would create their accounts in this case. Would an admin need to create accounts for every single user? (don't think we need to fix this but something to keep in mind)

Reply:
No, all users (no auth or authorization required) can create account, but only admins are able to create accounts with admin role

Reply:
oh okay, missread it. That's good